### DISCUSSION TITLE
Fundamental Data for zipline is avaiable with alpha-compiler

### DISCUSSION THREAD

**Discussion by carstenf** [stefan-jansen/zipline-reloaded#14] [Ideas]
Just to let you know, you can run Zipline with Fundamental Data, using 

https://github.com/pbharrin/alpha-compiler

I was using this for some while with zipline 1.3 and 1.4 and I works really great.
As well, it's easy to install and use.

installed it yesterday with zipline-reloaded and so far it looks good

**Comment by EcoFin** [comment]
I will take a look.  Thanks for the tip!

ay
